tests/countries/test_andorra.py

```diff
@@ -104,46 +105,46 @@ def test_2023(self):
 
         # AD-02, Canillo.
         self.assertHoliday(
-            Andorra(subdiv="AD-02", years=2023),
-            "2023-07-21",
-            "2023-07-22",
-            "2023-07-23",
+            Andorra(subdiv="02", years=2023),
```

Comment:
Just FYI:
```suggestion
            Andorra(subdiv="02"),
```
Would look cleaner (yeah, I know I wrote this couple months ago :) )